intro-HPC/ch_interactive_gent.tex

```diff
@@ -0,0 +1,54 @@
+\chapter{HPC-UGent interactive cluster}
```

Comment:
`debug` should also be in chapter title imho

Reply:
i don't agree. we really should try to sell it for its interactivity

Reply:
fine, whatever, just saying that "interactive cluster" is just wrong, we're not forcing only interactive jobs in any way, so it will filled up with short batch jobs in no time